activesupport/test/core_ext/date_and_time_behavior.rb

```diff
@@ -346,12 +346,24 @@ def test_before
     assert_equal true, date_time_init(2017, 3, 6, 12, 0, 0).before?(date_time_init(2017, 3, 7, 12, 0, 0))
   end
 
+  def test_on_or_before
```

Comment:
I tried to run the test locally, but kept getting no runs, so I just tried it in the console. This is what I was seeing 🤔:

```ruby
➜  rails git:(dvd-time-date-calc) cd activesupport
➜  activesupport git:(dvd-time-date-calc) bin/test test/core_ext/date_and_time_behavior.rb
Running 0 tests in a single process (parallelization threshold is 50)
Run options: --seed 52502

# Running:



Finished in 0.000094s, 0.0000 runs/s, 0.0000 assertions/s.
0 runs, 0 assertions, 0 failures, 0 errors, 0 skips
```

Reply:
The issue here is that `date_and_time_behavior` is not itself a test `Class`, its a `Module` that gets included in other test `Class`es to share the tests.